pkg/config/config.go

```diff
@@ -24,6 +24,11 @@ type Serialized struct {
 	Impersonate             string   `json:"impersonate"`
 	Gather                  []string `json:"gather"`
 	EnableGlobalObfuscation bool     `json:"enableGlobalObfuscation"`
+	Ocm                     struct {
```

Comment:
why is this defined as inline struct if you have this as full script on line 75? :-)

Reply:
Ohh good catch! Thank you.

Reply:
Well hmmm...this is slightly different type than the `OCMConfig` (which is embedded in `Controller` type). I can delete the `OCMConfig` type and define the attributes directly in the `Controller` type....I guess I don't have any better idea right now.